axum/src/serve.rs

```diff
@@ -158,6 +131,49 @@ where
     }
 }
 
+#[derive(Debug, Copy, Clone)]
+struct TowerToHyperService<S> {
+    service: S,
+}
```

Comment:
Would be nice if this was part of an axum-independent utility crate (it would be applicable for stuff like the caniuse.rs xtask as well, right?)

Reply:
Yeah my plan is to upstream it to hyper-utils.